### DISCUSSION TITLE
200-amp Current Transformer source on Ebay

### DISCUSSION THREAD

**Discussion by ICPete** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#157] [General]
I finally found a source for a current transformer rated for 200 amps and having the proper output:

[YHDC SCT-019TS](https://www.ebay.com/itm/283439437925?mkcid=16&mkevt=1&mkrid=711-127632-2357-0&ssspo=gGOtwrX_SQu&sssrc=2047675&ssuid=vpPY4xaISOC&widget_ver=artemis&media=COPY)

![image](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/assets/136994470/f1842a4d-c99e-4e82-99e8-32113df9b2a5)

The output of this component is 33 mA max at 200 amps input current, so it's perfect for the CircuitSetup 6-channel monitor.
It doesn't come with a plug, so you'll need to purchase a 3.5mm audio plug.

I just bought the last four of these that I found on Amazon (sorry!):

[Amazon listing for SCT-019TS](https://www.amazon.com/gp/product/B08H8671RP/ref=ppx_od_dt_b_asin_title_s00?ie=UTF8&psc=1)
[Note that the Amazon vendor is also shipping from China, and the Ebay vendor lists a somewhat faster delivery time to the US. The price was about the same from both sources.]

I needed to measure the 200-amp main breaker current at two places: at my service entrance panel and at my Protected Load Panel ("PLP", for a battery-backed-up photovoltaic system).

This SCT-019TS is openable to install it around the wire, but rather than being split and hinged, it has a small removable section of the core. I haven't received these yet, but when I do I'll upload some photos and describe my experience installing them. They are about 2-inches square, so maybe a bit more compact than the SCT-024 size sensor, yet still their opening is 19mm. For reference, THHN copper wire in 2/0 AWG is about 13mm diameter, and 4/0 AWG is about 16mm diameter. I have the latter at my PLP, so the 19mm opening should be fine.

**Comment by CircuitSetup** [comment]
Glad you found one that will work! Make sure to calibrate to the meter. These look very similar to some Magnalab CTs that I picked up early on (except they were voltage output).
